packages/create-gatsby/src/index.ts

```diff
@@ -2,17 +2,18 @@ import Enquirer from "enquirer"
 import cmses from "./cmses.json"
 import styles from "./styles.json"
 import features from "./features.json"
-import { initStarter, getPackageManager } from "./init-starter"
+import { initStarter, getPackageManager, gitSetup } from "./init-starter"
 import { installPlugins } from "./install-plugins"
 import c from "ansi-colors"
 import path from "path"
 import fs from "fs"
 import { plugin } from "./components/plugin"
 import { makePluginConfigQuestions } from "./plugin-options-form"
-import { center, rule, wrap } from "./components/utils"
+import { center, wrap } from "./components/utils"
 import { stripIndent } from "common-tags"
 import { trackCli } from "./tracking"
 import crypto from "crypto"
+import { reporter } from "./reporter"
```

Comment:
There was merge conflict here due to neighbouring
```
import { setSiteMetadata } from "./site-metadata"
```

added by https://github.com/gatsbyjs/gatsby/pull/28324 which is not backported and was merged before this PR